pkgs/tools/networking/openssh/common.nix

```diff
@@ -168,8 +168,11 @@ stdenv.mkDerivation {
     "sysconfdir=\${out}/etc/ssh"
   ];
 
-  passthru.tests = {
-    borgbackup-integration = nixosTests.borgbackup;
+  passthru = {
```

Comment:
We should definitely add a test with kerberos to passthru

Reply:
A NixOS test? We are not in a worse place than what we currently have on `master` and I am probably not the best person to write this test and to maintain it

https://github.com/NixOS/nixpkgs/pull/302688#discussion_r1563165964